.gitignore

```diff
@@ -4,7 +4,6 @@
 *.swn
 katello-installer*
 foreman-installer*
-puppet-*
```

Comment:
@ehelms This line matched one of my file templates; is there a more specific pattern to ignore? Or I could force not ignoring my file.